TestRun - Password Criteria Checker/Program.cs

```diff
@@ -0,0 +1,175 @@
+Console.WriteLine("         PASSWORD CRITERIA CHECKER         ");
+Console.WriteLine("-------------------------------------------");
+Console.WriteLine("");
+
+// criteria1
+// Begins testing of criteria1.
+// Adds 1 to criteriaCounter, indicating that there is now 1 additional rule to comply to.
+criteriaCounter++;
+
+// initiates variables denoting the minimum and maximum number of characters in the given password.
+int passLengthMin = 9;
+int passLengthMax = 25;
+// initiates an integer that measures the length of the given password.
+int passLength = userPass.Length;
+// intiates a boolean that confirms the length of the given password to be greater than or equal to the minimum, and less than or equal to the maximum.
+bool passLengthCheck = (passLength >= passLengthMin && passLength <= passLengthMax);
+// according to resulting true or false regard the last boolean check, outputs to console.
+// if true, adds 1 to complianceCounter to indicate passing score within this criteria
+if (passLengthCheck == true ) {
+    complianceCounter++;
+    Console.WriteLine(criteria1 + " : PASS");
+}
+else 
+{ Console.WriteLine(criteria1 + " : FAIL"); }
+
+// criteria2
+// Begins testing of criteria2.
+// Adds 1 to criteriaCounter, indicating that there is now 1 additional rule to comply to.
+criteriaCounter++;
+// initiates a boolean that confirms if any special characters exist within the given password.
+bool passSpecChar = (
+   userPass.Contains("!") ||
+   userPass.Contains("@") ||
+   userPass.Contains("#") ||
+   userPass.Contains("$") ||
+   userPass.Contains("%") ||
+   userPass.Contains("^") ||
+   userPass.Contains("&") ||
+   userPass.Contains("*") ||
+   userPass.Contains("(") ||
+   userPass.Contains(")") ||
+   userPass.Contains("-") ||
+   userPass.Contains("_") ||
+   userPass.Contains("=") ||
+   userPass.Contains("+") ||
+   userPass.Contains("[") ||
+   userPass.Contains("{") ||
+   userPass.Contains("]") ||
+   userPass.Contains("}") ||
+   userPass.Contains("\\") ||
+   userPass.Contains("|"));
+
+// according to results, outputs to console.
+// if true, adds 1 to complianceCounter to indicate passing score within this criteria.
+if (passSpecChar == true){
+    complianceCounter++;
+    Console.WriteLine(criteria2 + " : PASS");
+}
+else
+{
+    Console.WriteLine(criteria2 + " : FAIL");
+}
+
+// criteria3
+// begins testing of criteria3
+// Adds 1 to criteriaCounter, indicating that there is now 1 additional rule to comply to.
+criteriaCounter++;
+// initiates a boolean that confirms if any numerical characters are found within the given password.
+bool passNumChar =
+                    (
+   userPass.Contains("1")
+   || userPass.Contains("2")
+   || userPass.Contains("3")
+   || userPass.Contains("4")
+   || userPass.Contains("5")
+   || userPass.Contains("6")
+   || userPass.Contains("7")
+   || userPass.Contains("8")
+   || userPass.Contains("9")
+   || userPass.Contains("0")
+                    );

```

Comment:
Same idea here, but you could just do a regex like `/\d/` which matches on digits